### DISCUSSION TITLE
Tags implementation

### DISCUSSION THREAD

**Discussion by johnwc** [elsa-workflows/elsa-core#3785] [Q&A]
I see within the advanced tab of a workflow definition, that there is a field labeled `Tag`. How do we filter the workflow definitions by this tag in the workflow definition browser? We're using v2 of Elsa.

**Comment by sfmskywalker** [comment]
Hi @johnwc , that Tag feature was originally designed to just store a custom string value to associate with the workflow definition, but it would make perfect sense to be able to filter by tag in the definition browser! Please feel free to submit a feature request.

**Reply by johnwc** [reply]
@sfmskywalker  Is a feature request handled as a discussion post or issue creation?

**Reply by sfmskywalker** [reply]
As an issue would be good 👍🏻
